bench/bench/bench_mlp.py

```diff
@@ -18,14 +33,17 @@
 
 
 def _query_gpu_specs():
-    import subprocess
-    cmd = ["nvidia-smi", "--query-gpu=name", "--format=csv,noheader", "-i=0"]
-    output = subprocess.check_output(cmd, stderr=subprocess.DEVNULL).decode().strip()
-    name = output.splitlines()[0]
-    return {
-        "NVIDIA H100 80GB HBM3": {"MAX_TFLOPS8": 1979, "MAX_TFLOPS16": 989, "MAX_TBPS": 3.35}, "HGX GB200":
-        {"MAX_TFLOPS8": 4500, "MAX_TFLOPS16": 2250, "MAX_TBPS": 8.0}
-    }[name]
+    if not is_hip():
+        import subprocess
+        cmd = ["nvidia-smi", "--query-gpu=name", "--format=csv,noheader", "-i=0"]
+        output = subprocess.check_output(cmd, stderr=subprocess.DEVNULL).decode().strip()
+        name = output.splitlines()[0]
+        return {
+            "NVIDIA H100 80GB HBM3": {"MAX_TFLOPS8": 1979, "MAX_TFLOPS16": 989, "MAX_TBPS": 3.35}, "HGX GB200":
+            {"MAX_TFLOPS8": 4500, "MAX_TFLOPS16": 2250, "MAX_TBPS": 8.0}
+        }[name]
+    else:
+        return None
```

Comment:
NIT: Cleaner to use an early return here
```suggestion
    if is_hip():
        return None
        
    import subprocess
    cmd = ["nvidia-smi", "--query-gpu=name", "--format=csv,noheader", "-i=0"]
    output = subprocess.check_output(cmd, stderr=subprocess.DEVNULL).decode().strip()
    name = output.splitlines()[0]
    return {
        "NVIDIA H100 80GB HBM3": {"MAX_TFLOPS8": 1979, "MAX_TFLOPS16": 989, "MAX_TBPS": 3.35}, "HGX GB200":
        {"MAX_TFLOPS8": 4500, "MAX_TFLOPS16": 2250, "MAX_TBPS": 8.0}
    }[name]
```